cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/AbstractSchedulerService.java

```diff
@@ -81,7 +81,7 @@ protected final void startScheduler() {
       streamSizeScheduler.start();
       LOG.info("Started stream size scheduler");
     } catch (Throwable t) {
-      LOG.error("Error starting stream size scheduler {}", t.getCause(), t);
+      LOG.error("Error starting stream size scheduler", t);
```

Comment:
Interesting here: you already catch all Throwables. Here could wrap it into a SchedulerException instead of a RuntimeException.